targets/XPlatCppSdk/source/licenses/other dependencies.txt

```diff
@@ -0,0 +1,2 @@
+curl and jsoncpp may each have other sub-dependencies which all have additional licenses defined in nuget
```

Comment:
This file kindof doesn't even make sense anymore, because we're not using curl, and we're not using Nuget.